src/Models/SprintStory.php

```
@@ -30,8 +30,8 @@ class SprintStory
      */
     public static function assign(Database $db, int $sprintId, int $userStoryId): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)